### PR TITLE
js-eval: remove h>, simplify stuff

### PR DESCRIPTION
**Pull request by caub**
- Remove h> command, --harmony flag doesn't even exist anymore in node 14+
- We need to run code using node15, both in the main bot process and in docker if possible? because using `require('timers/promise')`
- I'll update babel plugins also in a 2nd commit

### CODE REVIEW

**Comment by brigand** [comment]
Let's keep the bot compatible with 12.x for now. I should upgrade the server to 14.x at some point, but not going to do odd majors.

**Comment by caub** [comment]
I'll update the code in jsEvalPlugin so, ok

File: src/plugins/js-eval/jsEvalPlugin.js [hunk 1]
```diff
@@ -33,23 +29,20 @@ module.exports = async function jsEvalPlugin({
 
   const hasMaybeTLA = /\bawait\b/.test(code);
 
-  if (mode === 'b' && !hasMaybeTLA) {
-    code = (await babel.transformAsync(code, { plugins: transformPlugins })).code;
```
Comment (caub) [inline src/plugins/js-eval/jsEvalPlugin.js:37]: done a fix, because that code above was not using parserOpts: {plugins: parserPlugins} before

**Comment by caub** [comment]
Ok done, the docker image can use node15 right?

File: src/plugins/js-eval/jsEvalPlugin.js (hunk 1 above)
Comment (caub) [reply src/plugins/js-eval/jsEvalPlugin.js:37]: this should allow us to use records and tuples, or other babel syntax things, but I'm not totally sure

**Comment by caub** [comment]
Hmm for record&tuple, only syntax support was added https://github.com/babel/babel/pull/10865, it's not transpiled to anything yet https://repl.it/@caub/babel-record-and-tuple#index.js, so we can't evaluate it, maybe engine262 has it though

**Comment by caub** [comment]
Guys, Record and Tuple is not ready yet currently, it'll be soon https://github.com/bloomberg/record-tuple-polyfill

I'll make a different PR for that when the time will come

**Review by brigand — APPROVED** [review]

**Comment by brigand** [comment]
Thanks!

**Pull request merged**